SpiralCompiler/Syntax/Parser.cs

```diff
@@ -364,12 +364,18 @@ private Expression ParsePostfixExpression()
         return left;
     }
 
+    private string EscapeStringLiteral(string tokenStr)
+    {
+        // TODO: new line ...
+        return tokenStr.Substring(1, tokenStr.Length - 2);
```

Comment:
Why not do it in this PR? My suggestion is to introduce an `EscapeChar(char ch)`, then loop through this str, escape after \ using the function